gemrb/core/GUI/GameControl.cpp

```diff
@@ -59,6 +59,13 @@ static const ResRef TestSpell = "SPWI207";
 
 uint32_t GameControl::DebugFlags = 0;
 
+static Actor* GetMainActor()
+{
+	if (auto actor = core->GetFirstSelectedPC(false))
+		return actor;
+	return core->GetGame()->selected[0];
```

Comment:
Cleaner version:
```
auto actor = core->GetFirstSelectedPC(false);
if (actor) return actor;
return core->GetFirstSelectedActor();
```

Reply:
I actually disagree with a name that has `PC` in it as that implies the actor is a PC. It could just be `GetActorToPerformAction` or something along those lines.

Reply:
or `GetActorToCommand`

Reply:
My suggestion doesn't have PC in it for the same reason.

Reply:
I clearly should have read the line below the code :)